recipe/meta.yaml

```diff
@@ -268,6 +274,7 @@ about:
   license_family: Apache
```

Comment:
might need to change the `home url` from `http` to `https`
home:https://opencv.org/

Reply:
Done!